### DISCUSSION TITLE
help with creating react-query like caching library for zustand

### DISCUSSION THREAD

**Discussion by pencil-user** [pmndrs/zustand#682] [Q&A]
I am wondering would would be a best approach for something like this. For one, what would you use to create keys that are objects to store different caches?

**Comment by matheusf31** [comment]
Hey, any update on that?

**Reply by hrishikeshjoshi91** [reply]
I m also struggling with right approach to integrate react query with zustand and 

Any help would be helpful

**Reply by agodinhost** [reply]
as far as I know Zustand approach is agnostic right? it´s intended to be used in any other javascript framework - react or no react.
React-query is tied to React (duuh, silly me),
so I guess that you wan't to follow this very same architecture - I'm right?
